mash_scripts/foreman/1.24.0/foreman-client-1.24-fedora28.mash

```diff
@@ -0,0 +1,12 @@
+[foreman-client-1.24-fedora28]
```

Comment:
F28 is EOL. Do we really want to build packages for this? I think we also dropped it in nightly configs some time ago.

Reply:
ACK, i'll remove it